tools/GraphClust/CollectResultsNoAlign/glob_report_no_align.xml

```diff
@@ -0,0 +1,184 @@
+<tool id="glob_report_no_align" name="cluster_collection_report_no_align" version="0.5" >
+  <requirements>
+    <requirement type="package" version="0.6.0">graphclust-wrappers</requirement>
+    <requirement type="package" version='0.5'>perl-array-utils</requirement>
+    <requirement type="package" version='0.18.1'>scikit-learn</requirement>
+    <requirement type="package" version='1.8.10'>locarna</requirement>
+    <requirement type="package" version='2.1'>rnaz</requirement>
+    <requirement type="package" version="1.1.2">infernal</requirement>
+    <requirement type="package" version='2.2.10'>viennarna</requirement>
+    <requirement type="package" version='1.3.30'>graphicsmagick</requirement>
+    <requirement type="package" version='0.6.1'>rscape</requirement>
+    <requirement type="package" version='6.0'>unzip</requirement>
+  </requirements>
+  <stdio>
+    <exit_code range="1:" />
+  </stdio>
+  <command>
+    <![CDATA[
+        unzip $FASTA  &> /dev/null &&
+
+          mkdir ./CMSEARCH &&
+          mkdir ./MODEL &&
+
+        #set $inputFiles = ""
+
+        #for $cms_res in $cmsearch_results:
+            ###set $inputFiles += str($cms_res.element_identifier)+','
+          ln -f -s  '$cms_res' ./CMSEARCH/$cms_res.element_identifier &&
+        #end for
+        #set $inputFiles = $inputFiles[:-1]
```

Comment:
is that still needed?

Reply:
good question. doesn't seem to be right?